pkg/controllers/options/tagging_controller.go

```diff
@@ -21,14 +21,20 @@ import (
 // TaggingControllerOptions contains the inputs that can
 // be used in the tagging controller
 type TaggingControllerOptions struct {
-	Tags      map[string]string
-	Resources []string
+	Tags       map[string]string
+	Resources  []string
+	RateLimit  float64
+	BurstLimit int
 }
 
 // AddFlags add the additional flags for the controller
 func (o *TaggingControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringToStringVar(&o.Tags, "tags", o.Tags, "Tags to apply to AWS resources in the tagging controller, in a form of key=value.")
 	fs.StringArrayVar(&o.Resources, "resources", o.Resources, "AWS resources name to add/remove tags in the tagging controller.")
+	fs.Float64Var(&o.RateLimit, "tagging-controller-rate-limit", o.RateLimit,
```

Comment:
Would `--tagging-controller-qps-limit` be a clearer name? Given it's a per-second value. Also clarify this in the flag description.

Reply:
Yup, will make this change. Seems like we have the following flags for Kubernetes:
`--kube-api-burst int32` and `--kube-api-qps float32` so I can rename these flags to something similar.

Reply:
As we discussed offline, `qps` might actually not be representative of the rate because the rate at which the controller processes items might not be the same at which it calls the service (especially if/when we add batching).

Hence, I am going to keep the same flag name but add more details to the description.